src/stream/src/executor/backfill/no_shuffle_backfill.rs

```diff
@@ -436,12 +473,13 @@ where
     }
 
     #[try_stream(ok = Option<StreamChunk>, error = StreamExecutorError)]
-    async fn snapshot_read(
-        upstream_table: &StorageTable<S>,
+    async fn snapshot_read<'a>(
```

Comment:
We may add some doc to this method showing that what'll be remaining in the `builder` and what will be outputted.